python/samples/kernel-syntax-examples/openai_function_calling.py

```diff
@@ -34,13 +32,6 @@
     ),
 )
 
-skills_directory = os.path.join(__file__, "../../../../samples/skills")
```

Comment:
This is an intentional removal of this skill configuration?

Reply:
yes, more of an oversight that it was in there in the first place, it is not usefull for the sample, so should be deleted to make sure the sample is as easy as possible. Could be a seperate PR...